package/src/utilities/makeKelvinLinearRGB.ts

```diff
@@ -0,0 +1,37 @@
+type Float3 = [number, number, number]
+
+// Converts sRGB to linear light
+function toLinear(c: number): number {
+  'worklet'
+  return c <= 0.04045 ? c / 12.92 : ((c + 0.055) / 1.055) ** 2.4
+}
+
+// Converts a Kelvin temp to linear sRGB color
+export default function makeKelvinLinearRGB(kelvin: number): Float3[] {
```

Comment:
I actually do not like much this function, it kind of lacks precision when converting from Kelvin, I'll take a look into it later